config/buildspec_profiler_sagemaker_pytorch_1_6_0_integration_tests.yml

```diff
@@ -0,0 +1,41 @@
+# Build Spec for AWS CodeBuild CI
+
+version: 0.2
+env:
+  variables:
+    override_smdebug: "true" # should we install the latest version of smdebug or use the existing version of smdebug?
+    enable_smdataparallel_tests: "true"
+    force_run_tests: "false"
+phases:
+  install:
+    commands:
+      # If we are not force running the tests, determine whether to run the tests based on the files changed in the branch compared to master.
+      - if [ "$force_run_tests" = "true" ]; then run_tests="true"; else chmod +x config/check_changed_files.sh && run_tests=`./config/check_changed_files.sh pytorch` ; fi
+      - apt-get update
+      - apt-get install sudo -qq -o=Dpkg::Use-Pty=0
+      - sudo apt-get install unzip -qq -o=Dpkg::Use-Pty=0
+      - pip install --upgrade pip==19.3.1
```

Comment:
I wonder if we can simply manage trivial dependencies like pip with a single file or env variable.

We can hold this thought for later; but it is clear that managing versions across multiple files is a certain pain point.

Reply:
There's this tool called dependabot I've used before that will automatically make pull requests for out of date dependencies. https://dependabot.com/

I can set this up in all of our repos. That way we won't have to spend time doing this ourselves.

Reply:
Awesome!